### DISCUSSION TITLE
В Edge не работают горячие клавиши

### DISCUSSION THREAD

**Discussion by vadash** [Chimildic/YaMuTools#46] [Вопрос/ответ] [answered]
![image](https://user-images.githubusercontent.com/1397582/163732227-729dce7b-4390-4c63-8782-970129f6fb0d.png)
Это я гдето ошибся или в edge это не работает ?

**Comment by Chimildic** [comment, marked as answer]
У расширения YaMuTools нет функций, связанных с горячими клавишами. Оно не управляет плеером.
